sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc

```diff
@@ -74,6 +78,10 @@ Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector) provides fu
 [[Config_Oauth]]
 == OAuth SAML Bearer Assertion Authentication
 
+=== Note
+
+OAuth SAML Bearer Assertion Authentication requires Minimum Mule EE Version 4.2.2.
```

Comment:
Mule 4.2.2 and later is required for OAuth SAML bearer assertion authentication.

Reply:
This is more accurate